crates/webidl/src/util.rs

```diff
@@ -18,7 +18,7 @@ pub(crate) fn shared_ref(ty: syn::Type) -> syn::Type {
     syn::TypeReference {
         and_token: Default::default(),
         lifetime: None,
-        mutability: None,
+        mutability: Some(syn::token::Mut::default()),
```

Comment:
Oh I think this isn't quite where we'll want to do this because this makes *everything* a mutable reference. In general `&mut SomeJsType` doesn't actually make sense in that the instance of `SomeJsType` isn't being changed (the pointer we're using at least). The instance itself may be modified but that's sort of just the normal story with interior mutability in Rust. 

To that end I think we'll want a dedicated new type like `Uint8ArrayMut` which would be the only type that uses `&mut`